### PR TITLE
List supported platforms in SDK docs v5 -Closes #1108

### LINKED ISSUE #1108: Mention supported platforms
Mention the supported platforms of the Lisk SDK (macOS & Ubuntu).

Mention especially, that the Lisk SDK does not run on Windows.

### PR DESCRIPTION
**Pull request by chris529**
Closes #1108

### CODE REVIEW

**Review by Tschakki — CHANGES_REQUESTED** [review]
The note should be added in the Quickstart and Setup guides.

Might be good to create the note as a partial.

Also please make sure to mention the specific versions of operating systems which are supported.

And provide the information in an easy to read way, bullet points would be nice for example.

**Comment by chris529** [comment]
Hey Mona! Good points, I think it looks better now.

File: modules/ROOT/pages/guides/app-development/setup.adoc [hunk 1]
```diff
@@ -23,6 +23,26 @@ If you prefer to not use Lisk Commander to get started with development, the <<m
...
+=== Operating Systems
+
+The Lisk Commander will run on both the Ubuntu and MacOS operating systems shown below:
+
+Ubuntu +
+
+* 18.04.03 (LTS) +
+* 18.04.04 (LTS) +
+* 18.05.05 (LTS) +
+* 20.04.02 (LTS) +
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/app-development/setup.adoc]:
```suggestion
* 18.04 (LTS)
* 20.04 (LTS)
```

no need to mention patch versions here

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -66,6 +66,26 @@ Each library is designed to be compatible with the xref:{url_protocol}[Lisk Prot
...
+=== Operating Systems
+
+The Lisk SDK will run on both the Ubuntu and MacOS operating systems shown below:
+
+Ubuntu +
+
+* 18.04.03 (LTS) +
+* 18.04.04 (LTS) +
+* 18.05.05 (LTS) +
+* 20.04.02 (LTS) +
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]:
```suggestion
* 18.04(LTS) 
* 20.04 (LTS) 
```

File: modules/ROOT/partials/install-node.adoc [hunk 3]
```diff
@@ -1,8 +1,8 @@
 === Node.js
 
 https://nodejs.org/[Node.js^] serves as the underlying engine for code execution.
-There are several different ways and version managers to install Node.js on your system.
-Please ensure that the latest version is used.
+There are several different ways and version managers to install Node.js on your system,
+and ensure that the latest version is used. As previously mentioned, please be aware that both the MacOS and Ubuntu operating systems are supported, and that the MS Windows operating system is not supported.
```
Comment (Tschakki) [inline modules/ROOT/partials/install-node.adoc]: please remove this sentence, it is not really fitting here.

File: modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -66,6 +66,26 @@ Each library is designed to be compatible with the xref:{url_protocol}[Lisk Prot
 
 == Usage
 
+=== Operating Systems
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]: This whole section should e moved to the Quickstart guide.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/partials/install-node.adoc (hunk 3 above)
Comment (chris529) [reply modules/ROOT/partials/install-node.adoc]: OK, now done 👍

File: modules/ROOT/pages/index.adoc (hunk 4 above)
Comment (chris529) [reply modules/ROOT/pages/index.adoc]: OK, now done 👍

File: modules/ROOT/partials/install-node.adoc [hunk 5]
```diff
@@ -1,8 +1,8 @@
 === Node.js
 
 https://nodejs.org/[Node.js^] serves as the underlying engine for code execution.
-There are several different ways and version managers to install Node.js on your system.
-Please ensure that the latest version is used.
+There are several different ways and version managers to install Node.js on your system,
+and ensure that the latest version is used.
```
Comment (Tschakki) [inline modules/ROOT/partials/install-node.adoc]: why this change? the sentence sounds weird to me like this.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/partials/install-node.adoc (hunk 5 above)
Comment (chris529) [reply modules/ROOT/partials/install-node.adoc]: Apologies, this was a typo error. I have now modified it 👍

File: modules/ROOT/pages/guides/app-development/setup.adoc [hunk 6]
```diff
@@ -23,6 +23,24 @@ If you prefer to not use Lisk Commander to get started with development, the <<m
 
 Lisk Commander offers various commands related to <<initializing-a-new-blockchain-application>> which can be used to conveniently get started with developing a blockchain application.
 
+=== Operating Systems
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/app-development/setup.adoc]:
```suggestion
=== Supported Operating Systems
```

File: modules/ROOT/pages/quickstart.adoc [hunk 7]
```diff
@@ -9,6 +9,25 @@
 
 The quickest way to bootstrap a blockchain application with the Lisk SDK is explained here.
 
+== Operating Systems
```
Comment (Tschakki) [inline modules/ROOT/pages/quickstart.adoc]:
```suggestion
== Supported Operating Systems
```

**Review by Tschakki — DISMISSED** [review]
LGTM! I left only a suggestion for the header title, which just came to my mind :)

**Pull request merged**